moderators.md

```diff
@@ -0,0 +1,143 @@
+merge the pull requests that comply with [R-multiverse official policies](policies.md),
+and either close the pull request (without merging) or ask for changes in non-compliant cases.
+
+As a [moderator](governance.md#moderator), you may optionally restrict your review to packages in your area of expertise.
+Simply search [contributions](https://github.com/r-multiverse/contributions/pulls) for relevant keywords,
+including [R-multiverse topics](https://r-multiverse.org/topics/).
+
+## Topics
+
+We encourage moderators to contribute [R-multiverse topics](https://r-multiverse.org/topics/)
+about their fields of expertise.
+To do so, submit a pull request to <https://github.com/r-multiverse/topics>^[
+Through triage access,
+R-multiverse [moderators](governance.md#moderator) are authorized to merge these pull requests.
+Please only edit the plain files without file extensions.
+All the HTML files are automatically generated.
+]
+to add a text file describing the subject matter ([example here](https://github.com/r-multiverse/topics/blob/main/bayesian)).
+A bot periodically rebuilds the website based on these text files.
+Seasoned [contributors](governance.md#contributor) will see your topic [listed here](https://r-multiverse.org/topics/)
+and add keywords to their [contributions](https://github.com/r-multiverse/contributions/pulls) accordingly.
+
+
+## Discussions
+
+[Moderators](governance.md#moderator) help answer questions on the [`r-multiverse/help`](https://github.com/r-multiverse/help)
+[issue](https://github.com/r-multiverse/help/issues) and [discussion](https://github.com/r-multiverse/help/discussions) forums.
+Questions may ask about specific packages, package reviews, the workings of R-multiverse, etc.
+A [moderator](governance.md#moderator) may close an
+[issue](https://github.com/r-multiverse/help/issues) or [discussion](https://github.com/r-multiverse/help/discussions) thread
+if it originally came from a [contributor](governance.md#contributor) or [user](governance.md#user)
+and the original issue is resolved.
+[Moderators](governance.md#moderator) should not close issues or discussions created by [administrators](team.md#administrators),
+including [official proposals](governance.md#proposals) to modify [R-multiverse policies](policies.md).
+
+## Enforcement
+
+[Moderators](governance.md#moderator) must understand [all R-multiverse policies](policies.md)
+and be prepared to enforce them.
+[Moderators](governance.md#moderator) must respond immediately and quickly to the following rare cases:
+
+1. If an R-multiverse package or [R-multiverse stakeholder](governance.md#r-multiverse-stakeholders) violates any applicable laws,
+contact the appropriate law enforcement agency.
+1. If an R-multiverse package on GitHub and violates any [GitHub policies](https://docs.github.com/en/site-policy),
+[report the violation to GitHub](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
```

Comment:
how would we keep track of individual reports?

Reply:
I hadn't thought of that. Should we? IIRC rOpenSci has a code of conduct violation reporting policy, right?

The mechanism for blocking a package is writing free-form text in the package listing to explain the situation. I did this recently with `DEPAHRI` because the package does not have a serious license and I did not hear back from the authors: https://github.com/r-multiverse/contributions/blob/main/packages/DEPAHRI. Do we need something more centralized, or something that doesn't require blocking a specific package?

Reply:
Would a public GitHub discussion be enough? I think our governance promises open communication, which would point us to https://github.com/r-multiverse/help/issues and/or https://github.com/r-multiverse/help/discussions. From https://github.com/r-multiverse/help/issues/77, it seems like @shikokuchuo may prefer not to make exceptions for these cases.

Reply:
Yes as long as this does not make the person who does the first report and tracks it a target of vengeance (a bit paranoid :sweat_smile:)

Reply:
Added 7c1c4ffe3f80652c3bd422d9fd64ef3304632a4a to edit the reporting guidelines:

> Each incident, the actions taken to respond to the incident, and the resolution must be reported as an issue at <https://github.com/r-multiverse/help/issues> with the label "policy violation".
If you do not feel comfortable taking direct action yourself, contact an [administrator](team.md#administrators) immediately and ask the [administrator](team.md#administrators) handle the situation.